src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -282,21 +282,43 @@ void CodegenNeuronCppVisitor::print_function_procedure_helper(const ast::Block&
     }
 }
 
-
-void CodegenNeuronCppVisitor::print_hoc_py_wrapper_function_body(
+void CodegenNeuronCppVisitor::print_hoc_py_wrapper_call_impl(
     const ast::Block* function_or_procedure_block,
     InterpreterWrapper wrapper_type) {
-    if (info.point_process && wrapper_type == InterpreterWrapper::Python) {
-        return;
-    }
     const auto block_name = function_or_procedure_block->get_node_name();
-    if (wrapper_type == InterpreterWrapper::HOC) {
-        printer->fmt_push_block("{}", hoc_function_signature(block_name));
+
+    const auto get_func_call_str = [&]() {
```

Comment:
This could be its own function. A docstring would also be useful.

Reply:
It feels like it's (unfortunately) tailor made for this situation. Notice how the arguments are not `a, b, c` but are grabbed from HOC, i.e. `getarg(1), getarg(2), getarg(3)`. That reduces the reusability quite a bit. We could try to make that part customizable, but we'll need to worry about `f(a + b, c)` which requires traversing the AST and printing as one goes.